dev/examples/data.js

```diff
@@ -554,7 +554,7 @@ const DATA = [
                     upperLimit: 30
                 },
                 y2: {
-                    show: false,
+                    show: true,
```

Comment:
Is there any negative consequence to other examples using this data object by changing these to `true`?

Reply:
Actually i added these changes to amke Jeremy work easy. Since he approved this PR i can goon and revert these chanegs.